content/syllabuses/web-dev.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Web Development
+---
+
+- Introduction to NCIT curriculum
+- NCIT 01: Conducting research and user interviews
+- NCIT 02: Reporting
+- NCIT 03: Research presentation
+- {{% contentlink "topics/agile-scrum" %}}
+- {{% contentlink "workshops/agile-scrum/" %}}
+- NCIT 18: Agile project management
+- NCIT 19: Agile meetings
+- {{% contentlink "workshops/how-to-be-a-professional" %}}
+- {{% contentlink "projects/linux/beginner" %}}
+- {{% contentlink "topics/web-dev-self-learning" %}}
+- {{% contentlink "topics/clean-code" %}}
+- {{% contentlink "projects/git-exercises" %}}
+- NCIT 04: Version control (GIT) for teamwork
+- {{% contentlink "projects/first-personal-website" %}}
+- {{% contentlink "topics/how-to-ask-for-help-with-your-code" %}}
+- {{% contentlink "workshops/big-picture-web-dev" %}}
+- NCIT 10: Workshop: Giving and receiving constructive feedback
+- NCIT 06: Web design for business
+- NCIT 07: Building an online business
+- NCIT 12: Business and technology
+- NCIT 09: Ethics and Professionalism in IT
+- {{% contentlink "projects/basic-flow-control-katas" %}} 
+- {{% contentlink "topics/js-and-node-specific/super-basic-intro-to-node" %}}
+- WORKSHOP: Live code. Super basic intro to node
+{{% contentlink "topics/unit-testing" %}}
+%}}: add some tests to existing code
```

Comment:
I'm not convinced this will render. 

Also unit testing is a prerequisite to many things so it's not necessary to explicitly link to it.